FX/BryanChi_FX Devices.lua

```diff
@@ -4500,13 +4532,13 @@
         function demo.PushStyle()
```

Comment:
The error came from the addition of new SeparatorText style vars in 0.8.4. The style editor code in the demo isn't meant to be copied in other scripts so it doesn't handle well future vars. The commit responsible for updating the demo for 0.8.4 is: https://github.com/cfillion/reaimgui/commit/5027834c0e3ddfd0271cc179605e71127e7fd38a#diff-f39b2c3b80f196dcc91c2638a61fad4452a3cf64aa09194dcdfe000dc7dae84eL2939. Specifically, it's the insertion of two new items in the `vec2` table in `GetStyleData`.

Other scripts can simply include demo.lua and use its style editor directly. No need to duplicate the code:

```lua
local demo = dofile(reaper.GetResourcePath() .. '/Scripts/ReaTeam Extensions/API/ReaImGui_Demo.lua')
local ctx = reaper.ImGui_CreateContext('My script')
local function loop()
  demo.PushStyle(ctx)
  demo.ShowDemoWindow(ctx)
  if reaper.ImGui_Begin(ctx, 'Dear ImGui Style Editor') then
    demo.ShowStyleEditor(ctx)
    reaper.ImGui_End(ctx)
  end
  demo.PopStyle(ctx)
  reaper.defer(loop)
end
reaper.defer(loop)
```

Reply:
Thanks for getting back so quick!
Looks like I was using old versions and that's why I wasn't experiencing any crashes.
I just updated and just by removing line 4469  -   r.ImGui_PopStyleVar(ctx, #cache['StyleVar']) , seems to have fixed it.

I do have some custom entries in the style editor so I guess I can't use the function as how you demonstrated?